java/gazelle/private/javaparser/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ go_library(
         "//java/gazelle/private/java",
         "//java/gazelle/private/javaparser/netutil",
         "//java/gazelle/private/javaparser/proto/gazelle/java/javaparser/v0:javaparser",
+        "//java/gazelle/private/sorted_set",
```

Comment:
Can we also correct the visibility here to `//java/gazelle:__subpackages__`? We choose to name it private after other bazel project instead of internal and so gazelle_go does not help.